### PR TITLE
WaiverDB processor.

### PR DESCRIPTION
**Pull request by ralphbean**
See also #450.

### CODE REVIEW

**Comment by bowlofeggs** [comment]
It seems like rebasing this PR on top of #451 would help its tests to pass.

File: fedmsg_meta_fedora_infrastructure/tests/__init__.py [hunk 1]
```diff
@@ -66,6 +66,7 @@
 from fedmsg_meta_fedora_infrastructure.tests.pdc import *
 from fedmsg_meta_fedora_infrastructure.tests.mbs import *
 from fedmsg_meta_fedora_infrastructure.tests.centos_ci import *
+from fedmsg_meta_fedora_infrastructure.tests.waiverdb import *
```
Comment (bowlofeggs) [inline fedmsg_meta_fedora_infrastructure/tests/__init__.py]: I recommend against this style of importing, as it's easy to end up with clashes, and even outside of clashes you can end up with a very polluted namespace. Explicit is better than implicit.

**Comment by ralphbean** [comment]
Rebase on top of #451 completed.

**Comment by ralphbean** [comment]
Can anyone help merge this one?

**Comment by pypingou** [comment]
Let's merge :)

**Pull request merged**